packages/vanilla/src/sass/components/_modal.scss

```diff
@@ -1,146 +1,69 @@
-//@use '../base/colors' as *;
-
-// Future CSS to use for hidden components
-.cbp-hidden {
-  display: none;
-}
-
-/* this is added to the body when a dialog is open */
-.has-modal {
-  overflow: hidden;
-}
-
-.cbp-modal__backdrop {
-  background-color: rgba(0, 0, 0, 0.3);
-  display: none;
-  position: fixed;
-  overflow-y: auto;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
-}
-
-.cbp-modal__backdrop.active {
-  display: block;
-}
-
-.cbp-modal, [role="dialog"] {
-  position: fixed;
-  left: 50%;
-  bottom: 2rem;
-  transform: translateX(-50%);
+.cbp-modal,
+dialog,
```

Comment:
I would not include the `dialog` tag or `role="dialog"` here because there are other uses than for a full-screen dialog/modal that may be affected.

Reply:
removed